__tests__/app.test.js

```diff
@@ -31,4 +31,25 @@ describe("Nc news API tests", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id", () => {
+    it("should respond with the article that matches the article_id passed in", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((res) => {
+          const article = res.body.article;
+          expect(article).toEqual(
+            expect.objectContaining({
+              article_id: 1,
+              title: "Living in the shadow of a great man",
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              topic: expect.any(String),
+            })
+          );
```

Comment:
you would also need tests for if someone requested a valid ID that didn't exist in the db (e.g. /api/articles/10000) and if someone made an incorrect request (e.g. /api/articles/banana)